ding_dibs.info

```diff
@@ -7,4 +7,5 @@ files[] = ding_dibs.module
 dependencies[] = dibs
 dependencies[] = ding_base
 dependencies[] = ding_provider
+dependencies[] = htmlmail
```

Comment:
The htmlmail module is not part of the code base. Please include it in the make file.